### PR TITLE
{tools}[GCCcore/11.3.0] remake v4.3+dbg-1.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by branfosj — DISMISSED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/8d52af7b44e235db6706752985bc66d6 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3147.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/c05dcfefcfb8741899857788f8824239 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
alvis-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/d5352b0e02a8b6796e72b77e7ebb504d for a full test report.

File: easybuild/easyconfigs/r/remake/remake-4.3+dbg-1.6-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/rocky/remake/releases/download/remake-%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['f6a0c6179cd92524ad5dd04787477c0cd45afb5822d977be93d083b810647b87']
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
```
Comment (Micket) [inline easybuild/easyconfigs/r/remake/remake-4.3+dbg-1.6-GCCcore-11.3.0.eb:19]:
```suggestion

dependencies = [
    ('libreadline', '8.1.2'),
]

```

**Review by Micket — CHANGES_REQUESTED** [review]
build issues are readline related. should use our libreadline instead.

File: easybuild/easyconfigs/r/remake/remake-4.3+dbg-1.6-GCCcore-11.3.0.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/r/remake/remake-4.3+dbg-1.6-GCCcore-11.3.0.eb:19]: i'm just gonna commit it so that i can run the tests

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/60df10a5017c652d7e51ff70bf79aae8 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**